cms/djangoapps/contentstore/views/item.py

```diff
@@ -791,7 +791,7 @@ def _move_item(source_usage_key, target_parent_usage_key, user, target_index=Non
         else:
             try:
                 target_index = int(target_index) if target_index is not None else None
-                if len(target_parent.children) < target_index:
+                if target_index is None or len(target_parent.children) < target_index:
```

Comment:
With Nonetype `<` fails in in python3.

Reply:
Just maintained the same behaviour in python2 and python3 for this `if` condition.